src/Browscap/Generator/Helper/BuildHelper.php

```diff
@@ -177,7 +179,7 @@ public static function run(
                         ->renderSectionBody($section, $collection, $sectionsWithVersion, $sectionName)
                         ->renderSectionFooter($sectionName);
 
-                    $output[$sectionName] = $sectionName;
+                    $output[$sectionName] = 1;
```

Comment:
This `$output` variable is only ever used on line 162 to see if the array key exists. I figured it would save a little bit of memory to not store the same value as the key and value in the array if only the key is ever used.